### DISCUSSION TITLE
Error in r$status_code : $ operator is invalid for atomic vectors

### DISCUSSION THREAD

**Discussion by yeszhu88** [MRCIEU/TwoSampleMR#481] [General]
I was trying to query data by running 
bmi2014_exp_dat <- extract_instruments(outcomes = 'ieu-a-2')
 
the error message of 
Error in r$status_code : $ operator is invalid for atomic vectors

Anyone help? Where is the problem and how to fix it? thaks.

**Comment by lixiaodong1999** [comment]
library(ieugwasr)

**Reply by BaofengWu55555** [reply]
I have tired for this,but still not work.

**Reply by would1920** [reply]
I worked! That's amazing

**Reply by SmokerX** [reply]
holy shit, that's magic!

**Reply by Norns740532** [reply]
That's amazing!!! Thank you!!!

**Reply by yanhuayizha** [reply]
Amazing，you are so good!

**Reply by yeszhu88** [reply]
I tried today but still didn't work.  I was working on the examples on this website: https://mrcieu.github.io/TwoSampleMR/articles/exposure.html  and stopped at:  ieugwasr::[api_status](https://rdrr.io/pkg/ieugwasr/man/api_status.html)()
 with error message: 
> ieugwasr::api_status()
Error: $ operator is invalid for atomic vectors

Anybody try it again to see if it still works or provide an example code for me to try?  Thanks a million

**Comment by BaofengWu55555** [comment]
Sometimes it's the network error, I tried for another day,the issue dismissed.
